_posts/2019-03-06-cockpit-189.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 189
+author: Gundersanne
+date: 2019-3-06
+tags: cockpit linux
+slug: cockpit-189
+category: release
+summary: Cockpit with Machines and Services improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 189.
+
+### Machines: Import existing image when creating VM
+
+Choose an existing disk image when create a new VM.
+
+![Create VM with existing image](/images/machines-existing-disk-image.png)
+
+### Machines: Introduce virtual networks
+
+Introduce read-only statistics for virtual networks.
+
+![Virtual networks in machines](/images/machines-virtual-networks.png)
+
+_Thanks to Simon Kobyda for this feature!_
+
+### Services: Filtering of services
+
+Services can now be filtered on name, description, and state. With the filter
+based on state, the 3 tables (one for each state) have been merged into one
+table.
```

Comment:
Perhaps this?

> Services can now be filtered using text. Results can be filtered additionally by name, description, and state. 
>
> In addition, service states have been merged into one list. The switcher buttons at the top have also been changed to tabs to make switching more obvious.

Reply:
Well it's not just text, the state filtering is a dropdown.

Reply:
Right; which is why I added "Results can be filtered additionally by name, description, and state."

But yeah, it could be wordsmithed further.

Reply:
But the name/description filtering is text and the state filtering is a dropdown, no? This makes it seem like the state/description/state filter is in addition to a text filter. While the text filter is the name/description filter.

Reply:
It defaults to all, and the input is to filter text across all the fields.